### PR TITLE
Test crowd funding

### PR DESCRIPTION
**Pull request by MathieuDutSik**

### CODE REVIEW

**Comment by ma2bd** [comment]
@MathieuDutSik After starting a GraphQL service (aka "node service"), one can only use the GraphQL client, not the CLI any more. This is a very easy error to make. @christos-h is going to add a lock to prevent this: https://github.com/linera-io/linera-protocol/issues/704

**Comment by ma2bd** [comment]
Also the app is not really usable without #634 (which was closed by mistake)

File: examples/fungible/src/lib.rs [hunk 1]
```diff
@@ -9,6 +9,8 @@ use std::collections::BTreeMap;
 /// An operation.
 #[derive(Deserialize, Serialize)]
 pub enum Operation {
+    /// Obtain the total balance of the accounts
+    TotalBalance { },
```
Comment (ma2bd) [inline examples/fungible/src/lib.rs:13]: Did you mean to add a query?

Comment (MathieuDutSik) [reply examples/fungible/src/lib.rs:13]: Yes, I wanted to have the possibility to retrieve the total balance. This is to make the control of the accounts more stringent, that is that there was no money in other accounts. But I had some problem to make it work completely, so I should have eliminated that part.

Comment (ma2bd) [reply examples/fungible/src/lib.rs:13]: Well, for one it should be in the `enum Query`

**Comment by ma2bd** [comment]
Was incorporated in #720 
Thanks!

**Pull request closed without merging**